tailwind.config.js

```diff
@@ -1,6 +1,9 @@
 /** @type {import('tailwindcss').Config} */
 export default {
-	darkMode: 'selector',
```

Comment:
CRAAZY!! Amazing IA, I love this! I will test and review ASAP.

Reply:
my name is perhaps, most likely not IA